Source/WebCore/rendering/GlyphDisplayListCache.cpp

```diff
@@ -126,6 +126,8 @@ DisplayList::DisplayList* GlyphDisplayListCache::getDisplayList(const LayoutRun&
     }
 
     if (auto displayList = font.displayListForTextRun(context, textRun)) {
+        if (displayList->items().isEmpty())
+            return nullptr;
```

Comment:
This probably needs some explanation of how we may end up with an empty list and why it is not unexpected.